src/templates/index.jade

```diff
@@ -4,22 +4,23 @@ html
         meta(charset='utf-8')
         <link rel="icon" type="image/png" href="public/img/favicon-32.png" sizes="32x32">
         <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+        include metatags.jade
```

Comment:
`metatarsi` is too broad. Please rename it to something like `socialmetatags` etc.